restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java

```diff
@@ -86,10 +94,6 @@ void testDefaultOpts() {
         assertEquals(def.isUseNativeTransports(), options.isUseNativeTransports());
         assertEquals(def.getConnectorThreads(), options.getConnectorThreads());
         assertEquals(def.getIoThreads(), options.getIoThreads());
-        assertEquals(def.getBizThreadsCore(), options.getBizThreadsCore());
-        assertEquals(def.getBizThreadsMax(), options.getBizThreadsMax());
-        assertEquals(def.getBizThreadsBlockingQueueLength(), options.getBizThreadsBlockingQueueLength());
-        assertEquals(def.getBizThreadsKeepAliveTimeSeconds(), options.getBizThreadsKeepAliveTimeSeconds());
```

Comment:
plz do not remove existing unit tests of deprecated functions

Reply:
thx, have reverted!